Sources/StreamChatUI/CommonViews/SendButton/CooldownView.swift

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+/// A button for showing a cooldown when Slow Mode is active.
+open class CooldownView: _View, AppearanceProvider {
+    public private(set) lazy var cooldownLabel: UILabel = UILabel()
```

Comment:
Let's follow the pattern we have in other UI components with `Content` type:
```suggestion
    public struct Content {
        public var cooldown: Int
    }
    
    open var content: Content? {
        didSet { updateContentIfNeeded() }
    }
    
    public private(set) lazy var cooldownLabel: UILabel = UILabel()
    
    open override func updateContent() {
       super. updateContent()
       
       cooldownLabel.text = ...
    }
    
```